thirdparty/dns/JSDNS.jsm

```diff
@@ -625,24 +636,23 @@ function DNS_ReverseIPHostname(ip) {
 	return q[3] + "." + q[2] + "." + q[1] + "." + q[0] + ".in-addr.arpa";
 }
 
-function queryDNSRecursive(server, host, recordtype, callback, callbackdata, hops, servers) {
+function queryDNSRecursive(servers, host, recordtype, callback, callbackdata, hops) {
 	"use strict";
 	
-	// if more when one server is given
-	if (servers !== undefined) {
+	let server;
+	let serverObj = null;
+	
+	// if more than one server is given
+	if (typeof servers === "object") {
+		log.debug("DNS: More servers: "+Object.entries(servers));
```

Comment:
Fine with the additional logging, just want to make sure it is intended and not just a leftover of some debugging you did.

Reply:
Indeed, there were two left overs ;)